src/sql-config/config/base/013_Invoicing.sql

```diff
@@ -0,0 +1,177 @@
+#parse("src/sql-config/config/vars.vm")
+-- import global variables
+$vars_shop_supplier
+$vars_decision_beans
+$vars_execution_beans
+
+-- PAUSE velocity parser
+#[[
+
+-- local variables go here
+shopref bigint;
+communicationPartner bigint;
+documentType int;
+-- end variables block with 
+BEGIN
+
+-- dedicated invoice number range per channel
+IF NOT EXISTS (SELECT NULL FROM "InvoicingNoConfigDO" WHERE "shopRef" = shop_intronics_b2c AND "numberRangeFormatterDefRef" = 5)
+THEN
+	INSERT INTO "InvoicingNoConfigDO"(id, "name", "countGenerated", "creationDate", enabled, 
+				"generationDate","startNumber","endNumber", "numberRangeFormatterDefRef", 
+				"invoicingTypeDefRef","lastGeneratedNumber", "maxNotAllocatedInvoiceNo", 
+				"modificationDate", "startDate","shopRef", version)
+	    SELECT 	nextval('"InvoicingNoConfigDO_id_seq"'), 'InvoiceNo config - inTRONICS B2C', 0, CURRENT_TIMESTAMP, true,
+	    		null, '1000000000', '2999999999', 500, 
+	    		null, null, 5, 
+	    		CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, shop_intronics_b2c, 1;
+END IF;
+
+IF NOT EXISTS (SELECT NULL FROM "InvoicingNoConfigDO" WHERE "shopRef" = shop_intronics_b2b AND "numberRangeFormatterDefRef" = 5)
+THEN
+	INSERT INTO "InvoicingNoConfigDO"(id, "name", "countGenerated", "creationDate", enabled, 
+				"generationDate","startNumber","endNumber", "numberRangeFormatterDefRef", 
+				"invoicingTypeDefRef","lastGeneratedNumber", "maxNotAllocatedInvoiceNo", 
+				"modificationDate", "startDate","shopRef", version)
+	    SELECT 	nextval('"InvoicingNoConfigDO_id_seq"'), 'InvoiceNo config - inTRONICS B2B', 0, CURRENT_TIMESTAMP, true,
+	    		null, '3000000000', '4999999999', 500, 
```

Comment:
NumberRangeFormatterDefDO=500.
I guess the correct value is 5 as I do not see an extended range formatter type in project.